<!-- Make sure launch ctrl_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="thewalkingdead" type="task10.py" name="task10" output="screen"/>
</launch>